tests/parallel_test_core.py

```diff
@@ -45,7 +56,7 @@ def main():
   cores = int(os.environ.get('EMCC_CORES') or multiprocessing.cpu_count())
   pool = multiprocessing.Pool(processes=cores)
   args = [[x] + sys.argv[1:] for x in optimal_order]
-  filenames = pool.map(run_mode, args, chunksize=1)
+  num_failures = pool.map(run_mode, args, chunksize=1)
```

Comment:
is this summing the return codes? doesn't that assume they are 0 or 1, and not anything else?